### PR TITLE
deploying ubuntu image to docker hub

### PR DESCRIPTION
**Pull request by kzidane**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,8 +3,14 @@ services: docker
 script: docker build . -t cs50/cli
 before_deploy: docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
 deploy:
-  provider: script
+- provider: script
   script: docker push cs50/cli
+  on:
+    branch: master
```
Comment (dmalan) [inline .travis.yml]: Should be `ubuntu` branch!

* `cs50/cli` and `cs50/cli:latest` should be based on `master` branch.
* `cs50/cli:ubuntu` should be based on `ubuntu` branch.

File: .travis.yml [hunk 2]
```diff
@@ -3,8 +3,14 @@ services: docker
...
+- provider: script
+  script: docker tag cs50/cli cs50/cli:ubuntu; docker push cs50/cli:ubuntu
```
Comment (dmalan) [inline .travis.yml]: These shouldn't be the same!

File: .travis.yml (hunk 1 above)
Comment (kzidane) [reply .travis.yml]: There are two deployment providers, this one is for master (pushes cs50/cli:latest), and https://github.com/cs50/cli/pull/20/files#diff-354f30a63fb0907d4ad57269548329e3R13 is for ubuntu (pushes cs50/cli:ubuntu).

File: .travis.yml (hunk 2 above)
Comment (kzidane) [reply .travis.yml]: The build step is the same for all branches. It builds the image as cs50/cli. This just ensures the image is tagged ubuntu before pushing.

File: .travis.yml (hunk 1 above)
Comment (dmalan) [reply .travis.yml]: Oh  phew, I see, thanks.

File: .travis.yml (hunk 2 above)
Comment (dmalan) [reply .travis.yml]: Just to be sure, this should definitely be `docker tag cs50/cli cs50/cli:ubuntu`, even though former typically implies `cs50/cli:latest`?

Comment (kzidane) [reply .travis.yml]: Yup. See https://github.com/cs50/cli/pull/20/files#diff-354f30a63fb0907d4ad57269548329e3R3!

**Pull request merged**